src/components/manage/Blocks/LeadImage/Edit.jsx

```diff
@@ -95,7 +98,7 @@ class Edit extends Component {
           />
         )}
         <SidebarPortal selected={this.props.selected}>
-          <LeadImageSidebar {...this.props} />
+          <LeadImageSidebar {...{ ...this.props, properties }} />
```

Comment:
```suggestion
          <LeadImageSidebar {...this.props} {...properties} />
```
This leads to the same result but it would be more readable IMHO